Library/Homebrew/test/dev-cmd/extract_spec.rb

```diff
@@ -23,12 +23,27 @@
       system "git", "add", "--all"
       system "git", "commit", "-m", "testball 0.2"
     end
+    return { name: target.name, path: path }
+  end
 
-    expect { brew "extract", "testball", target.name, "--version=0.1" }
+  it "retrieves the most recent version of formula" do
+    expect { brew "extract", "testball", target[:name] }
       .to be_a_success
+    expect(target[:path]/"Formula/testball@0.2.rb").to exist
+    expect(Formulary.factory(target[:path]/"Formula/testball@0.2.rb").version).to be == "0.2"
+  end
 
-    expect(path/"Formula/testball@0.1.rb").to exist
+  it "retrieves the specified version of formula" do
```

Comment:
`retrieves the specified version of formula, defaulting to most recent`

Reply:
I've split the original test case into 2 test cases. One for `specified version`, another for `defaulting to most recent`. The latter case didn't get tested in the original single test case.

Reply:
Ok.